### PR TITLE
[bot] Documentation formatting

### PR DESCRIPTION
**Pull request by emacspace**
Merge with care - I'm just a stupid bot. Beep boop.

### CODE REVIEW

File: layers/+lang/c-c++/README.org [hunk 1]
```diff
@@ -244,9 +244,9 @@ Additionally, if you have =c-c++-enable-google-newline= variable set then
 set that up like this:
 
 #+BEGIN_SRC emacs-lisp
-  (c-c++ :variables
-         c-c++-enable-google-style t
-         c-c++-enable-google-newline t)
+ (c-c++ :variables
+ c-c++-enable-google-style t
+ c-c++-enable-google-newline t)
```
Comment (sdwolfz) [inline layers/+lang/c-c++/README.org]: Hmm... this looks wrong. I would have expected it to suggest removing only one space from the beginning of all 3 lines. @JAremko?

Comment (JAremko) [reply layers/+lang/c-c++/README.org]: @sdwolfz  That's because someone forgot `#+END_SRC` :shipit:    

Don't think it is autofixable :thinking:

Comment (sdwolfz) [reply layers/+lang/c-c++/README.org]: Dammit, I was specifically looking for this last night but missed it multiple times. For some reason that block if code did not stand out to me.

**Comment by sdwolfz** [comment]
I'll cherry-pick add the missing `#+END_SRC` this evening.

**Comment by sdwolfz** [comment]
Amended and cherry-picked into develop branch: 63bf38217abda2d875ca326b0bf1e9475d887a30.

**Pull request closed without merging**